circuit_maintenance_parser/parsers/verizon.py

```diff
@@ -0,0 +1,81 @@
+"""Verizon parser."""
+import logging
+import re
+from typing import Dict
+from dateutil import parser
+from bs4.element import ResultSet  # type: ignore
+
+from circuit_maintenance_parser.errors import ParsingError
+from circuit_maintenance_parser.parser import Html, Impact, CircuitImpact, Status
+
+logger = logging.getLogger(__name__)
+
+# pylint: disable=too-many-branches
+
+
+class HtmlParserVerizon1(Html):
+    """Notifications Parser for Verizon notifications."""
+
+    def parse_html(self, soup, data_base):
+        """Execute parsing."""
+        data = data_base.copy()
+        try:
+            logger.debug("PARSING")
```

Comment:
parsing what? maybe a more detailed message would help :)

Reply:
That was only temporary, but will keep and add more detail if that works. Can remove if required.